test/integration-tests/Sudoswap.test.ts

```diff
@@ -2,61 +2,60 @@ import { CommandType, RoutePlanner } from './shared/planner'
 import SUDOSWAP_ABI from './shared/abis/Sudoswap.json'
 import { ERC721, UniversalRouter, Permit2 } from '../../typechain'
 import { resetFork } from './shared/mainnetForkHelpers'
-import { ALICE_ADDRESS, DEADLINE } from './shared/constants'
+import { ALICE_ADDRESS, DEADLINE, ETH_ADDRESS, SUDOLETS_721, SUDOLETS_PAIR, SUDOLETS_ROUTER } from './shared/constants'
 import deployUniversalRouter, { deployPermit2 } from './shared/deployUniversalRouter'
 import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
 import hre from 'hardhat'
 import { BigNumber } from 'ethers'
 import { abi as ERC721_ABI } from '../../artifacts/solmate/src/tokens/ERC721.sol/ERC721.json'
 import { expect } from 'chai'
+import { parseEther } from 'ethers/lib/utils'
+import { MSG_SENDER } from '@uniswap/router-sdk'
```

Comment:
I wonder if it's just better to pull this in on line 5 from this repo, mostly bc this just changed but hasn't been released yet  😶